HW03/ihar_shkuryn/ihar_shkuryn_hw_03_t_03.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'time'
+
+class Homework3
+  PATTERN = /Calling core with action:/
+  def task3(log)
+    arr = log.split("\n")
```

Comment:
could you give more descriptive variable names. The array doesn't say anything about what's in there (it applies to everything else too)
Try to give descriptive names to variables that will reflect what you put in them.
try to avoid `I`, `x` and act variables naming